public/app/paywall.js

```
@@ -429,9 +429,12 @@ const Paywall = (() => {
 
       // Production: call backend — price resolution happens server-side via env vars
       try {
+        const headers = { 'Content-Type': 'application/json' };
+        const authToken = (typeof window !== 'undefined' && window.Auth?.currentUser?.token) || null;
```

ESLint

[error] 'window' is not defined.

(no-undef)